docs/source/release/v6.9.0/diffraction.rst

```diff
@@ -10,59 +10,59 @@ Powder Diffraction
+- Added the ``UpdateInstrumentParameters`` property in :ref:`AddSampleLog <algm-AddSampleLog>` which updates the instrument parameters after adding the log. This allows for instruments with detectors on motors (e.g. SNAP) to update the geometry when adding logs.
+- A new toolbar button named "Hide Legend" is introduced to toggle the visibility of the legend box of the :ref:`Fitting tab <ui engineering fitting>`. By Default the button is unchecked.
 
 Bugfixes
 ############
-- Stop plotting data with different data units to plot on Engineering diffraction fitting tab
-- Resolved the out of sync observation between the fit algorithm parameters returned by the :ref:`Fitting tab <ui engineering fitting>` of the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` and the Parameters table after completing a fitting.
-- Validate the characters entered in the RB Number box in :ref:`Engineering Diffraction GUI<Engineering_Diffraction-ref>` prior to creating a custom directory structure to write the artifacts in Calibration, Focus, and GSAS II tabs.
-- Avoid cropping the range of TOF data in the GSAS-II tab of :ref:`Engineering Diffraction GUI<Engineering_Diffraction-ref>` from its original TOF range
-- Updated the dmax value used to generate ``Pawley Refinement`` from ``3 Ang`` into ``4.2 Ang`` since ENGIN-X has wavelength range of upto ``6 Ang``.
-- Avoid writing invalid background estimation parameters in the Engineering Diffration's table of fitting tab.
-- When invalid parameters are entered for the background estimation in the :ref:`Fitting tab <ui engineering fitting>` of the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` they are overwritten with the last valid parameters used when the background is next calculated (immediately if `Subtract BG` is checked in the table).
-- Avoid crashing the engineering diffraction GUI :ref:`Engineering Diffraction GUI<Engineering_Diffraction-ref>` when errors are raised from GSASIIscriptable module in the GSAS-II tab.
+- Prevented the plotting of data with different units on the same plot in the :ref:`Fitting tab <ui engineering fitting>`.
+- Resolved the out of sync observation between the fit algorithm parameters returned by the :ref:`Fitting tab <ui engineering fitting>` and the Parameters table after completing a fit.
+- Added validation of the characters entered in the RB Number box in the :ref:`Engineering Diffraction interface <Engineering_Diffraction-ref>` prior to creating a custom directory structure to write the artifacts in Calibration, Focus, and GSAS II tabs.
+- Cropping the range of TOF data in the GSAS-II tab of the :ref:`Engineering Diffraction interface <Engineering_Diffraction-ref>` from its original TOF range is now avoided.
+- Updated the dmax value used to generate ``Pawley Refinement`` from ``3 Ang`` into ``4.2 Ang`` since ENGIN-X has a wavelength range of up to ``6 Ang``.
```

Comment:
This is extremely minor so feel free to disregard it if you just want this merged, but using this will insert the actual angstrom symbol which looks a bit neater. 
```suggestion
- Updated the dmax value used to generate ``Pawley Refinement`` from 3 :math:`\mathrm{\AA{}}` into 4.2 :math:`\mathrm{\AA{}}` since ENGIN-X has a wavelength range of up to 6 :math:`\mathrm{\AA{}}`.
```